lib-platform/util/fileutil.c

```diff
@@ -64,6 +64,18 @@ fileutil_fsync(FILE * fp, const char * name)
 		goto err0;
 	}
 
+#ifdef F_FULLFSYNC
+	/*
+	 * MacOS-specific "ask the drive to flush all buffered data".
+	 * Not supported on all filesystems.  Even on supported filesystems,
+	 * some FireWire drives are known to ignore this request.
+	 */
+	if (fcntl(fd, F_FULLFSYNC)) {
+		warnp("fcntl");
```

Comment:
Since this is not supported on all filesystems, we'll probably get scary EINVAL/ENOTSUPP/etc errors here.  Sinec this is just advisory, let's just make the attempt and ignore the return code from `fcntl`.